cli/input/input.go

```diff
@@ -69,7 +69,7 @@ func ConfirmTx(w io.Writer, tx *transaction.Transaction) error {
 	if err != nil {
 		return err
 	}
-	if 0 < len(ln) && ln[0] == 'y' || ln[0] == 'Y' {
+	if 0 == len(ln) || ln[0] == 'y' || ln[0] == 'Y' {
```

Comment:
This makes "yes" a default, which is not what we want.

Reply:
Fixed.